src/goto-symex/symex_main.cpp

```diff
@@ -544,6 +544,16 @@ void goto_symext::run_intrinsic(
     // Already modelled in builtin_libs
     return;
   }
+  else if(has_prefix(symname, "c:@F@__ESBMC_parity"))
```

Comment:
If we remove the prefix `__ESBMC_`, we don't need these two if-statements.

Reply:
I'll do it.